kratos/includes/lock_object.h

```diff
@@ -23,102 +24,94 @@
 
 // Project includes
 
+
 namespace Kratos
 {
-  ///@addtogroup KratosCore
-  ///@{
+///@addtogroup KratosCore
+///@{
 
-  ///@name Kratos Classes
-  ///@{
+///@name Kratos Classes
+///@{
 
-  /// This class defines stores a lock and gives interface to it.
-  /** The class makes a tiny wrapper over the ahared memory locking mechanisms
-  */
-  class LockObject
-    {
-    public:
-      ///@name Life Cycle
-      ///@{
+/// This class defines stores a lock and gives interface to it.
+/** The class makes a tiny wrapper over the sahared memory locking mechanisms
```

Comment:
still a spelling error 
```suggestion
/** The class makes a tiny wrapper over the shared memory locking mechanisms
```